### ISSUE TITLE
build failure on mvn clean install: maven-javadoc-plugin:2.9.1 API incompatibility

### ISSUE THREAD

**Issue by KonradHoeffner** [SmartDataAnalytics/Sparqlify#94]
Following the readme, I cloned the repository and did `mvn clean install`, which results in:

    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project sparqlify-parent: Execution attach-javadocs of goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar failed: An API incompatibility was encountered while executing org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar: java.lang.ExceptionInInitializerError: null

## Environment

```
$ mvn -v
Apache Maven 3.8.4 (9b656c72d54e5bacbed989b64718c159fe39b537)
Maven home: /opt/maven
Java version: 18, vendor: N/A, runtime: /usr/lib/jvm/java-18-openjdk
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.17.3-arch1-1", arch: "amd64", family: "unix"

$ java --version
openjdk 18 2022-03-22
OpenJDK Runtime Environment (build 18+0)
OpenJDK 64-Bit Server VM (build 18+0, mixed mode)

$ git rev-parse HEAD
ccb5c718ae86b58315775afb53c5b607f2f2009c
```

## Logs
```
/tmp$ git clone git@github.com:SmartDataAnalytics/Sparqlify.git
Cloning into 'Sparqlify'...
remote: Enumerating objects: 17531, done.
remote: Counting objects: 100% (1395/1395), done.
remote: Compressing objects: 100% (706/706), done.
remote: Total 17531 (delta 556), reused 1083 (delta 376), pack-reused 16136
Receiving objects: 100% (17531/17531), 39.19 MiB | 12.68 MiB/s, done.
Resolving deltas: 100% (9565/9565), done.
/tmp$ cd Sparqlify 
Sparqlify$ mvn clean install
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 21 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 6 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Sparqlify SPARQL->SQL rewriter - Parent                            [pom]
[INFO] obda-api-parent                                                    [pom]
[INFO] obda-api-core                                                      [jar]
[INFO] obda-api-r2rml                                                     [jar]
[INFO] obda-api-sml                                                       [jar]
[INFO] obda-api-bundle                                                    [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - Type System                       [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - Tests                             [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - Core                              [jar]
[INFO] obda-api-examples                                                  [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - Cli                               [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - Platform                          [war]
[INFO] Sparqlify SPARQL->SQL rewriter - Tests                             [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - JPA Mapper                        [jar]
[INFO] Sparqlify SPARQL->SQL rewriter - Web Admin Server                  [war]
[INFO] sparqlify-pkg-parent                                               [pom]
[INFO] sparqlify-pkg-deb-cli                                              [jar]
[INFO] Sparqlify - Debian Packaging - Tomcat Common                       [war]
[INFO] Sparqlify - Debian Packaging - Tomcat7                             [jar]
[INFO] Sparqlify - Debian Packaging - Tomcat8                             [jar]
[INFO] Sparqlify - Nexus Deploy Workaround                                [jar]
[INFO] 
[INFO] ----------------< org.aksw.sparqlify:sparqlify-parent >-----------------
[INFO] Building Sparqlify SPARQL->SQL rewriter - Parent 0.9.1            [1/21]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ sparqlify-parent ---
[INFO] 
[INFO] --- maven-javadoc-plugin:2.9.1:jar (attach-javadocs) @ sparqlify-parent ---
[WARNING] Error injecting: org.apache.maven.plugin.javadoc.JavadocJar
java.lang.ExceptionInInitializerError
    at org.apache.maven.plugin.javadoc.AbstractJavadocMojo.<clinit> (AbstractJavadocMojo.java:190)
    at jdk.internal.misc.Unsafe.ensureClassInitialized0 (Native Method)
    at jdk.internal.misc.Unsafe.ensureClassInitialized (Unsafe.java:1160)
    at jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized (MethodHandleAccessorFactory.java:300)
    at jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor (MethodHandleAccessorFactory.java:103)
    at jdk.internal.reflect.ReflectionFactory.newConstructorAccessor (ReflectionFactory.java:236)
    at java.lang.reflect.Constructor.acquireConstructorAccessor (Constructor.java:546)
    at java.lang.reflect.Constructor.newInstanceWithCaller (Constructor.java:496)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:483)
    at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance (DefaultConstructionProxyFactory.java:126)
    at com.google.inject.internal.ConstructorInjector.provision (ConstructorInjector.java:114)
    at com.google.inject.internal.ConstructorInjector.access$000 (ConstructorInjector.java:32)
    at com.google.inject.internal.ConstructorInjector$1.call (ConstructorInjector.java:98)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
    at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
    at com.google.inject.internal.ConstructorInjector.construct (ConstructorInjector.java:93)
    at com.google.inject.internal.ConstructorBindingImpl$Factory.get (ConstructorBindingImpl.java:306)
    at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
    at com.google.inject.internal.InjectorImpl.getInstance (InjectorImpl.java:1086)
    at org.eclipse.sisu.space.AbstractDeferredClass.get (AbstractDeferredClass.java:48)
    at com.google.inject.internal.ProviderInternalFactory.provision (ProviderInternalFactory.java:85)
    at com.google.inject.internal.InternalFactoryToInitializableAdapter.provision (InternalFactoryToInitializableAdapter.java:57)
    at com.google.inject.internal.ProviderInternalFactory$1.call (ProviderInternalFactory.java:66)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
    at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
    at com.google.inject.internal.ProviderInternalFactory.circularGet (ProviderInternalFactory.java:61)
    at com.google.inject.internal.InternalFactoryToInitializableAdapter.get (InternalFactoryToInitializableAdapter.java:47)
    at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
    at org.eclipse.sisu.inject.Guice4$1.get (Guice4.java:162)
    at org.eclipse.sisu.inject.LazyBeanEntry.getValue (LazyBeanEntry.java:81)
    at org.eclipse.sisu.plexus.LazyPlexusBean.getValue (LazyPlexusBean.java:51)
    at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:263)
    at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:255)
    at org.apache.maven.plugin.internal.DefaultMavenPluginManager.getConfiguredMojo (DefaultMavenPluginManager.java:520)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:124)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke (Method.java:577)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.StringIndexOutOfBoundsException: Range [0, 3) out of bounds for length 2
    at jdk.internal.util.Preconditions$1.apply (Preconditions.java:55)
    at jdk.internal.util.Preconditions$1.apply (Preconditions.java:52)
    at jdk.internal.util.Preconditions$4.apply (Preconditions.java:213)
    at jdk.internal.util.Preconditions$4.apply (Preconditions.java:210)
    at jdk.internal.util.Preconditions.outOfBounds (Preconditions.java:98)
    at jdk.internal.util.Preconditions.outOfBoundsCheckFromToIndex (Preconditions.java:112)
    at jdk.internal.util.Preconditions.checkFromToIndex (Preconditions.java:349)
    at java.lang.String.checkBoundsBeginEnd (String.java:4589)
    at java.lang.String.substring (String.java:2703)
    at org.apache.commons.lang.SystemUtils.getJavaVersionAsFloat (SystemUtils.java:1133)
    at org.apache.commons.lang.SystemUtils.<clinit> (SystemUtils.java:818)
    at org.apache.maven.plugin.javadoc.AbstractJavadocMojo.<clinit> (AbstractJavadocMojo.java:190)
    at jdk.internal.misc.Unsafe.ensureClassInitialized0 (Native Method)
    at jdk.internal.misc.Unsafe.ensureClassInitialized (Unsafe.java:1160)
    at jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized (MethodHandleAccessorFactory.java:300)
    at jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor (MethodHandleAccessorFactory.java:103)
    at jdk.internal.reflect.ReflectionFactory.newConstructorAccessor (ReflectionFactory.java:236)
    at java.lang.reflect.Constructor.acquireConstructorAccessor (Constructor.java:546)
    at java.lang.reflect.Constructor.newInstanceWithCaller (Constructor.java:496)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:483)
    at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance (DefaultConstructionProxyFactory.java:126)
    at com.google.inject.internal.ConstructorInjector.provision (ConstructorInjector.java:114)
    at com.google.inject.internal.ConstructorInjector.access$000 (ConstructorInjector.java:32)
    at com.google.inject.internal.ConstructorInjector$1.call (ConstructorInjector.java:98)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
    at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
    at com.google.inject.internal.ConstructorInjector.construct (ConstructorInjector.java:93)
    at com.google.inject.internal.ConstructorBindingImpl$Factory.get (ConstructorBindingImpl.java:306)
    at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
    at com.google.inject.internal.InjectorImpl.getInstance (InjectorImpl.java:1086)
    at org.eclipse.sisu.space.AbstractDeferredClass.get (AbstractDeferredClass.java:48)
    at com.google.inject.internal.ProviderInternalFactory.provision (ProviderInternalFactory.java:85)
    at com.google.inject.internal.InternalFactoryToInitializableAdapter.provision (InternalFactoryToInitializableAdapter.java:57)
    at com.google.inject.internal.ProviderInternalFactory$1.call (ProviderInternalFactory.java:66)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
    at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
    at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
    at com.google.inject.internal.ProviderInternalFactory.circularGet (ProviderInternalFactory.java:61)
    at com.google.inject.internal.InternalFactoryToInitializableAdapter.get (InternalFactoryToInitializableAdapter.java:47)
    at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
    at org.eclipse.sisu.inject.Guice4$1.get (Guice4.java:162)
    at org.eclipse.sisu.inject.LazyBeanEntry.getValue (LazyBeanEntry.java:81)
    at org.eclipse.sisu.plexus.LazyPlexusBean.getValue (LazyPlexusBean.java:51)
    at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:263)
    at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:255)
    at org.apache.maven.plugin.internal.DefaultMavenPluginManager.getConfiguredMojo (DefaultMavenPluginManager.java:520)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:124)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke (Method.java:577)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Sparqlify SPARQL->SQL rewriter - Parent 0.9.1:
[INFO] 
[INFO] Sparqlify SPARQL->SQL rewriter - Parent ............ FAILURE [  0.362 s]
[INFO] obda-api-parent .................................... SKIPPED
[INFO] obda-api-core ...................................... SKIPPED
[INFO] obda-api-r2rml ..................................... SKIPPED
[INFO] obda-api-sml ....................................... SKIPPED
[INFO] obda-api-bundle .................................... SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Type System ....... SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Tests ............. SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Core .............. SKIPPED
[INFO] obda-api-examples .................................. SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Cli ............... SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Platform .......... SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Tests ............. SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - JPA Mapper ........ SKIPPED
[INFO] Sparqlify SPARQL->SQL rewriter - Web Admin Server .. SKIPPED
[INFO] sparqlify-pkg-parent ............................... SKIPPED
[INFO] sparqlify-pkg-deb-cli .............................. SKIPPED
[INFO] Sparqlify - Debian Packaging - Tomcat Common ....... SKIPPED
[INFO] Sparqlify - Debian Packaging - Tomcat7 ............. SKIPPED
[INFO] Sparqlify - Debian Packaging - Tomcat8 ............. SKIPPED
[INFO] Sparqlify - Nexus Deploy Workaround ................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.803 s
[INFO] Finished at: 2022-04-20T09:55:27+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project sparqlify-parent: Execution attach-javadocs of goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar failed: An API incompatibility was encountered while executing org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar: java.lang.ExceptionInInitializerError: null
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.apache.maven.plugins:maven-javadoc-plugin:2.9.1
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/konrad/.m2/repository/org/apache/maven/plugins/maven-javadoc-plugin/2.9.1/maven-javadoc-plugin-2.9.1.jar
[ERROR] urls[1] = file:/home/konrad/.m2/repository/org/slf4j/slf4j-jdk14/1.5.6/slf4j-jdk14-1.5.6.jar
[ERROR] urls[2] = file:/home/konrad/.m2/repository/org/slf4j/jcl-over-slf4j/1.5.6/jcl-over-slf4j-1.5.6.jar
[ERROR] urls[3] = file:/home/konrad/.m2/repository/commons-cli/commons-cli/1.2/commons-cli-1.2.jar
[ERROR] urls[4] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-interactivity-api/1.0-alpha-4/plexus-interactivity-api-1.0-alpha-4.jar
[ERROR] urls[5] = file:/home/konrad/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
[ERROR] urls[6] = file:/home/konrad/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
[ERROR] urls[7] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.11/plexus-interpolation-1.11.jar
[ERROR] urls[8] = file:/home/konrad/.m2/repository/backport-util-concurrent/backport-util-concurrent/3.1/backport-util-concurrent-3.1.jar
[ERROR] urls[9] = file:/home/konrad/.m2/repository/org/apache/maven/reporting/maven-reporting-api/3.0/maven-reporting-api-3.0.jar
[ERROR] urls[10] = file:/home/konrad/.m2/repository/org/apache/maven/maven-archiver/2.5/maven-archiver-2.5.jar
[ERROR] urls[11] = file:/home/konrad/.m2/repository/org/apache/maven/shared/maven-invoker/2.0.9/maven-invoker-2.0.9.jar
[ERROR] urls[12] = file:/home/konrad/.m2/repository/org/apache/maven/shared/maven-common-artifact-filters/1.3/maven-common-artifact-filters-1.3.jar
[ERROR] urls[13] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-sink-api/1.0/doxia-sink-api-1.0.jar
[ERROR] urls[14] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-site-renderer/1.0/doxia-site-renderer-1.0.jar
[ERROR] urls[15] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-core/1.0/doxia-core-1.0.jar
[ERROR] urls[16] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-i18n/1.0-beta-7/plexus-i18n-1.0-beta-7.jar
[ERROR] urls[17] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-velocity/1.1.7/plexus-velocity-1.1.7.jar
[ERROR] urls[18] = file:/home/konrad/.m2/repository/org/apache/velocity/velocity/1.5/velocity-1.5.jar
[ERROR] urls[19] = file:/home/konrad/.m2/repository/oro/oro/2.0.8/oro-2.0.8.jar
[ERROR] urls[20] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-decoration-model/1.0/doxia-decoration-model-1.0.jar
[ERROR] urls[21] = file:/home/konrad/.m2/repository/commons-collections/commons-collections/3.2/commons-collections-3.2.jar
[ERROR] urls[22] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-module-apt/1.0/doxia-module-apt-1.0.jar
[ERROR] urls[23] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-module-fml/1.0/doxia-module-fml-1.0.jar
[ERROR] urls[24] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-module-xdoc/1.0/doxia-module-xdoc-1.0.jar
[ERROR] urls[25] = file:/home/konrad/.m2/repository/org/apache/maven/doxia/doxia-module-xhtml/1.0/doxia-module-xhtml-1.0.jar
[ERROR] urls[26] = file:/home/konrad/.m2/repository/commons-lang/commons-lang/2.4/commons-lang-2.4.jar
[ERROR] urls[27] = file:/home/konrad/.m2/repository/commons-io/commons-io/2.2/commons-io-2.2.jar
[ERROR] urls[28] = file:/home/konrad/.m2/repository/org/apache/httpcomponents/httpclient/4.2.3/httpclient-4.2.3.jar
[ERROR] urls[29] = file:/home/konrad/.m2/repository/org/apache/httpcomponents/httpcore/4.2.2/httpcore-4.2.2.jar
[ERROR] urls[30] = file:/home/konrad/.m2/repository/commons-codec/commons-codec/1.6/commons-codec-1.6.jar
[ERROR] urls[31] = file:/home/konrad/.m2/repository/commons-logging/commons-logging/1.1.1/commons-logging-1.1.1.jar
[ERROR] urls[32] = file:/home/konrad/.m2/repository/log4j/log4j/1.2.14/log4j-1.2.14.jar
[ERROR] urls[33] = file:/home/konrad/.m2/repository/com/thoughtworks/qdox/qdox/1.12.1/qdox-1.12.1.jar
[ERROR] urls[34] = file:/home/konrad/.m2/repository/junit/junit/3.8.1/junit-3.8.1.jar
[ERROR] urls[35] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-archiver/2.1.2/plexus-archiver-2.1.2.jar
[ERROR] urls[36] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-io/2.0.4/plexus-io-2.0.4.jar
[ERROR] urls[37] = file:/home/konrad/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] : Range [0, 3) out of bounds for length 2
```

**Comment by KonradHoeffner** [comment]
Upgrading the Apache Maven Javadoc Plugin to 2.10.4 (latest 2.x.y) does not fix this, however upgrading to 3.3.2 (latest version) does.

**Comment by Aklakan** [comment]
Should be fixed by now
